### PR TITLE
VAC-442 youtube markdown embeds

### PR DESCRIPTION
**Pull request by blackmjck**
## Description
Page help desperately wants for video capability, so now we can add YouTube embedded videos natively like so:
```![youtu.be/videoID]```

### What's included?
- Native iframe embeds added for youtube share links
- Ability to embed playlists as well as single videos

#### General Tests for Every PR

- [x] `npm run serve:prod` still works.
- [x] `npm run tslint` passes.
- [x] `npm run stylelint` passes.
- [x] `npm test` passes and code coverage is not lower.
- [x] `npm run build:lib` still works.

##### Screenshots or link to StackBlitz/Plunker
Screen Shot 2021-02-04 at 2 34 06 PM

### CODE REVIEW

**Comment by fs250044** [comment]
Should this be made more generic (e.g., `[video url]`) in order to more easily support other video types (Vimeo, custom MP4s, etc)?

**Comment by blackmjck** [comment]
**Note:** had to remove the `webdriver-manager update` part of the post-install hook as it's throwing errors

File: src/platform/markdown/markdown.component.ts [hunk 1]
```diff
@@ -295,4 +296,30 @@ export class TdMarkdownComponent implements OnChanges, AfterViewInit {
...
+      }
+      return `<iframe allowfullscreen frameborder="0" src="https://www.youtube.com/embed/videoseries?list=${id}"></iframe>`;
+    }
+
+    return html
+      .replace(ytLongWatch, convert)
+      .replace(ytLongEmbed, convert)
+      .replace(ytShort, convert)
+      .replace(ytPlaylist, convertPL);
+  }
```
Comment (JulieMarie) [inline src/platform/markdown/markdown.component.ts:324]: Maybe a better spot for this is in `markdown-utils/markdown-utils.ts`?

**Review by JulieMarie — CHANGES_REQUESTED** [review]
Thank you for startling me awake when I went to the demo....not sure `autoplay` is the way to go!  :P

File: src/platform/markdown/markdown.component.ts (hunk 1 above)
Comment (blackmjck) [reply src/platform/markdown/markdown.component.ts]: Good thinking. Done

**Review by JulieMarie — APPROVED** [review]

**Review by owilliams320 — APPROVED** [review]
LGTM

**Pull request merged**